patches/tock/07-firmware-protect.patch

```diff
@@ -0,0 +1,423 @@
+diff --git a/boards/components/src/firmware_protection.rs b/boards/components/src/firmware_protection.rs
```

Comment:
Is there an upstream pull-request to discuss the capsule/component/chip additions (which should stay in this patch as shortly as possible)?

Reply:
Not yet. I needed the feature for OpenSK first. Starting the PR for Tock is next but will clearly happen.